bentoml/marshal/marshal.py

```diff
@@ -178,6 +180,7 @@ def __init__(
             "or launch more microbatch instances to accept more concurrent connection.",
             self.CONNECTION_LIMIT,
         )
+        self._client = None
```

Comment:
We probably don't need to lazy initialize client here. Client is pretty much always needed, correct?

Reply:
> Client is pretty much always needed, correct?

Yeah. But IMO we best only do value assignments in the `__init__`.

In addition, in this case, an aiohttp client session should be initialized with a running asyncio event loop.
https://github.com/aio-libs/aiohttp/issues/3331